spec/integration/runners/queue/rspec_runner_spec.rb

```diff
@@ -327,6 +332,173 @@ def log_command_result(stdout, stderr, status)
     end
   end
 
+  context 'when rails_helper file does not exist' do
+    it 'does not require the rails_helper file when running RSpec' do
+      rspec_options = ''
+
+      spec_helper = <<~SPEC
+      require 'knapsack_pro'
+      KnapsackPro::Adapters::RSpecAdapter.bind
+      SPEC
+
+      spec_a = Spec.new('a_spec.rb', <<~SPEC)
+        describe 'A_describe' do
+          it 'A1 test example' do
+            expect(1).to eq 1
+          end
+        end
+      SPEC
+
+      spec_b = Spec.new('b_spec.rb', <<~SPEC)
+        describe 'B_describe' do
+          it 'B1 test example' do
+            expect(1).to eq 1
+          end
+        end
+      SPEC
+
+      spec_c = Spec.new('c_spec.rb', <<~SPEC)
+        describe 'C_describe' do
+          it 'C1 test example' do
+            expect(1).to eq 1
+          end
+        end
+      SPEC
+
+      generate_specs(spec_helper, rspec_options, [
+        [spec_a, spec_b],
+        [spec_c],
+      ])
+
+      actual = subject
+
+      expect(actual.stdout.scan(/--require rails_helper/).size).to eq 0
```

Comment:
```suggestion
      expect(actual.stdout).to_not include('--require rails_helper')
```

I think this would be simpler and more consistent with other examples. What do you think?